jac-uk/apply
# #1355 Updated Review page to correctly show answers related to Welsh posts

**Author checklist**

- ✅  Include primary ticket number in title - e.g. "#123 New styling for widget" - and any additional tickets in description
- ✅  Fill in the details below and delete as appropriate

---

## What's included?
Made small changes to Review component to correctly show answers related to Welsh posts/requirements.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
As a candidate apply for a vacancy with Welsh posts and Welsh requirements.
Ensure that candidate answers on the application Review page->Welsh posts section correctly reflect provided information.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Application Review page:

![Sc1](https://user-images.githubusercontent.com/40855898/122070808-2ca73c00-cdee-11eb-95d2-e7aa6f6f4114.png)


---
PREVIEW:DEVELOP
